libraries/AR_WPNav/AR_WPNav.h

```diff
@@ -62,7 +62,7 @@ class AR_WPNav {
     const Location &get_oa_destination() { return _oa_destination; }
 
     // return heading (in degrees) and cross track error (in meters) for reporting to ground station (NAV_CONTROLLER_OUTPUT message)
-    float wp_bearing_cd() const { return _wp_bearing_cd; }
```

Comment:
This 2nd one was not a bug actually.  It's not really in the mavlink spec but there are two mavlink messages that we take advantage of to show the user both the intermediate and final destination.  So the "POSITION_TARGET_GLOBAL_INT" message is used to send the intermediate position.  The "NAV_CONTROLLER_OUTPUT" is used to send the final position.

If necessary we can discuss with @tridge on whether we are doing the correct thing or not by using these two mavlink messages.